app/team.go

```diff
@@ -183,6 +183,24 @@ func (a *App) UpdateTeamScheme(team *model.Team) (*model.Team, *model.AppError)
 	return oldTeam, nil
 }
 
+func (a *App) UpdateTeamPrivacy(team *model.Team, user *model.User) (*model.Team, *model.AppError) {
```

Comment:
You are not using the user at all, probably we can remove that restriction from the whole command.

Reply:
Hi @jespino , thank you very much for the review!
I am adding the `--username` flag since the command description [here](https://github.com/mattermost/mattermost-server/issues/9670#issue-370524485) stated that the command should supply which user (username) modify the team privacy. Does this mean that we do not need the `--username` flag for this command?

Reply:
Hi @jespino , thank you very much for the review! 🙂 

I am adding the `--username` flag since the command description [here](https://github.com/mattermost/mattermost-server/issues/9670#issue-370524485) stated that this command needs to be supplied of which user (username) modified the team's privacy. Just to make sure, does this mean that we do not need to input any `--username` flag anymore for this particular command?

Reply:
Yes, I think is not needed, sorry the delay, I was on holidays.